### PR TITLE
Int Radices

### PR DESCRIPTION
**Pull request by mlutze**
Replacing https://github.com/flix/flix/pull/996 since it didn't like working with the squashed commit.

### CODE REVIEW

**Comment by magnus-madsen** [comment]
Let me know when this is ready for review.

**Comment by mlutze** [comment]
It's ready!

**Comment by magnus-madsen** [comment]
I have been busy, but I will try to get this reviewed soon.

File: main/src/ca/uwaterloo/flix/language/ast/ParsedAst.scala [hunk 1]
```diff
@@ -288,7 +288,7 @@ object ParsedAst {
       * @param lit  the int8 literal.
```
Comment (magnus-madsen) [inline main/src/ca/uwaterloo/flix/language/ast/ParsedAst.scala]: Update documentation for radix (also applies to other AST nodes).

**Comment by magnus-madsen** [comment]
Looks good to me. Just a few lines of documentation to ParsedAst.

**Comment by magnus-madsen** [comment]
Thanks !:)

**Pull request merged**